README.md

```diff
@@ -0,0 +1,36 @@
+# HOM
+
+The helpful old man!
+
+## Contributing
+
+Commands shown here are unix compatible and may need to be altered for Windows users.
+
+Create a new virtual environment and activate it
+
+```bash
+$ python -m venv .venv
+$ source ./.venv/bin/activate
+```
+
+Install dependencies
+
+```bash
+$ pip install -r requirements.dev.txt
+```
+
+Copy the example `.env` file
+
+```bash
+$ cp .example.env .env
+```
+
+Update the values in `.env` for your development bot
+
+Ensure the bot has the proper permissions and intents (all)
+
+Run the bot
+
+```bash
+$ python -m hom
+```
```

Comment:
After running the bot, one must run `!sync` to register the slash commands. I think it should be included in the readme. Else you'd have to read the source code to find this out.

Reply:
Should we just sync the commands on bot startup automatically without the need for the command?